indico/htdocs/sass/partials/_inputs.scss

```diff
@@ -123,6 +123,7 @@ span[contenteditable] {
         display: none;
 
         &:checked + label,
+        &:checked:disabled + label,
```

Comment:
i guess this is to keep the blue style for the selected horizontal radio buttons even when they are disabled? can you check e.g. with @OmeGak  if that will affect any other areas where we might not want this?